elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/EntityProjectionTranslator.java

```diff
@@ -28,10 +28,13 @@
 import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Objects;
+import java.util.Optional;
 import java.util.Set;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import javax.ws.rs.core.MultivaluedMap;
```

Comment:
I'm not sure we want this particular Map class (which is a JAX-RS dependency) to be required outside elide-core.   We may want RequestScope to keep the headers private and instead have a public method to ask for a particular header.